checker/checker.js

```diff
@@ -23,8 +23,23 @@ gets fired from github pr creation/update webhook.
 * if not signed, give an 'x' and tell them to go sign at https://opensource.adobe.com/cla.html
 */
 const valid_pr_events = ['opened', 'reopened', 'synchronize'];
+// if we're in a merge queue, then we had to pass our cla check already
+const valid_merge_group_events = ['checks_requested'];
 
 async function main (params) {
+  // technically, we shouldn't even need to check this here, as anything without a
+  // pull_request property should have been filtered out by the webhook
+  // which means that a merge_queue should already pass, the only reason we have been
+  // failing so far I believe is because we were not getting the event at all, so couldn't
+  // mark it as passing. On the chance I'm wrong, I'm leaving this here for now.
+  // but we may not even need a code change
```

Comment:
Further proof we don't need any changes, we just need to enable the events like we did in the cla-bot-playground https://github.com/adobe/cla-bot-playground/pull/370
This PR was merged based on the already existing cla-bot code, it would hang previous to changing the settings of the repo and branch protections.